### PR TITLE
gravitacia-dev

### PR DESCRIPTION
**Pull request by gravitacia**

### CODE REVIEW

**Comment by jbeder** [comment]
You have a lot of random files in this commit. Could you please only include the relevant changes?

**Comment by gravitacia** [comment]
> You have a lot of random files in this commit. Could you please only include the relevant changes?

fixed

**Comment by Osteri** [comment]
aa

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 build/
 /tags
 /bazel-*
+cmake-build-debug/
```
Comment (jbeder) [inline .gitignore:4]: Please drop this file's changes.

File: include/yaml-cpp/emitter.h [hunk 2]
```diff
@@ -177,7 +178,9 @@ inline Emitter& Emitter::WriteStreamable(T value) {
     }
   }
 
-  if (!special) {
+  if (static_cast<float>(static_cast<int>(value)) == value){
+    stream << std::fixed << std::setprecision(1) << value;
```
Comment (jbeder) [inline include/yaml-cpp/emitter.h:182]: I *think* I see what you're trying to do, but could you please add a test that demonstrates this change?

File: include/yaml-cpp/emitter.h [hunk 3]
```diff
@@ -177,7 +178,9 @@ inline Emitter& Emitter::WriteStreamable(T value) {
...
+  }else if (!special) {
```
Comment (jbeder) [inline include/yaml-cpp/emitter.h:183]: Missing space between `}` and `else`.

**Review by jbeder — CHANGES_REQUESTED** [review]
Please update the PR description to describe this change.

**Pull request closed without merging**